joomla/joomla-cms
# a11y improvement: plugins Pagenavigation

### Summary of Changes
The Pagenavigation plugin displays links to the previous and next article. The text of the links are the words "Prev" and "Next". They do not contain any information about the content/topic of the articles. For display users, tooltips with article titles have been added. For Assistive Technology users - aria-label with information that the link gives access to previous/next article and article title.
Developed by Web Accessibility Team.
### Testing Instructions
1. Start the article list page in one category.
2. Open the article from the list.
3. Set the mouse pointer over the links (buttons) Prev and Next. Tooltips should be displayed.
4. Run a screen reader, such as NVDA (or ChromoVox for Chrome). Move tabs between links. Check that the screen reader correctly announces the purpose of the links to previous and next article
### Expected result
1. Tooltips with linked article title are displayed above the Prev and Next buttons.
2. The screen reader will announce links to the titles of the articles to which they lead.
### Actual result
Currently, the user is not informed exactly about the purpose of the link. The content of the links "previous" and "next" is quite vague.
### Documentation Changes Required
1. No changes to the documentation required - no new functionality introduced.
2. You need to add two new language constants do administrator/language/en-GB/en-GB.en-GB.plg_content_pagenavigation.ini file
PLG_PAGENAVIGATION_PREV_A11Y="Access to the previous article: %s"
PLG_PAGENAVIGATION_NEXT_A11Y="Access to the next article: %s"
